dislib/trees/decision_tree.py

```diff
@@ -8,7 +8,7 @@
 from sklearn.tree import DecisionTreeClassifier as SklearnDTClassifier
```

Comment:
please consider exposing only objects that are necessary in dislib/regression/__init__.py 
(currently all of them are exposed)